bigchaindb/__init__.py

```diff
@@ -9,7 +9,7 @@
     'server': {
         # Note: this section supports all the Gunicorn settings:
         #       - http://docs.gunicorn.org/en/stable/settings.html
-        'bind': '0.0.0.0:9984',
+        'bind': 'localhost:9984',
```

Comment:
@vrde:

1. Could you briefly comment on why this change is necessary?
2. Is there a reason for not parametrizing the host?